src/qt/walletmodel.cpp

```diff
@@ -336,11 +331,6 @@ WalletModel::SendCoinsReturn WalletModel::sendCoins(WalletModelTransaction &tran
 {
     QByteArray transaction_array; /* store serialized transaction */
 
-    if(isAnonymizeOnlyUnlocked())
-    {
-        return AnonymizeOnlyUnlocked;
-    }
-
```

Comment:
Transaction is only created in WalletModel::prepareTransaction and wallet lock is verified there now, see 57c4c1b. I think this one was/is excessive and we should be fine here without additional check.